### PR TITLE
Add intervention

### PR DESCRIPTION
**Pull request by oscarwyatt**
Adds SaB intervention

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: app/controllers/concerns/sab_pages_testable.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+module SabPagesTestable
+  CUSTOM_DIMENSION = 65
```
Comment (oscarwyatt) [inline app/controllers/concerns/sab_pages_testable.rb:2]: Need to find out what dimension we can use

File: test/controllers/sab_ab_test.rb [hunk 2]
```diff
@@ -0,0 +1,88 @@
+require "test_helper"
+
+class ContentItemsControllerTest < ActionController::TestCase
+  include GovukAbTesting::MinitestHelpers
+  INTERVENTION_CSS_SELECTOR = "gem-c-intervention".freeze
+
+  test "shows intervention for variant B" do
```
Comment (oscarwyatt) [inline test/controllers/sab_ab_test.rb:7]: I think these can probably be refactored

File: app/views/content_items/guide.html.erb [hunk 3]
```diff
@@ -18,6 +18,10 @@
   <div class="govuk-grid-column-two-thirds">
     <%= render 'govuk_publishing_components/components/title', { title: @content_item.content_title } %>
 
+    <% if should_show_sab_intervention? %>
```
Comment (oscarwyatt) [inline app/views/content_items/guide.html.erb:21]: also probably ought to make these a one liner

**Review by maxf — APPROVED** [review]
lgtm

**Pull request merged**